diff -Paur ../orig-conf/my.cnf ./my.cnf
--- ../orig-conf/my.cnf	2011-11-16 21:32:20.662972300 +0000
+++ ./my.cnf	2011-11-16 21:35:44.251324800 +0000
@@ -120,6 +120,8 @@
 # ssl-cert=/etc/mysql/server-cert.pem
 # ssl-key=/etc/mysql/server-key.pem
 
+collation_server = utf8_unicode_ci
+character_set_server = utf8
 
 
 [mysqldump]
@@ -129,6 +131,7 @@
 
 [mysql]
 #no-auto-rehash	# faster start of mysql but no tab completition
+default-character-set = utf8
 
 [isamchk]
 key_buffer		= 16M
